index.html

```diff
@@ -250,6 +250,31 @@ <h3 class="text-3">LET’S CONNECT</h3>
 
 
     </main>
+    <footer class="footer">
+        <div class="foot-image">
+            <h3 class="contactme">
+                
+                Contact me 
+            </h3>
+            <p class="contact-info">
+                If you have an application you are interested in developing,
+                a feature that you need built or a project that needs coding.
+                I’d love to help with it
+            </p>
+            <form action="https://formspree.io/f/xnqrnyla" class="contact-form" method="POST">
+                <ul class="form-item">
+                    <li ><input type="text"  placeholder="username" maxlength="30" class="username" name="username" required></li>
+                   <li><input type="email"  placeholder="Enter email" name="email"  class="email" required></li> 
+                   <li ><textarea class="message" maxlength="500" placeholder="Write your message here"></textarea></li>
+                   <li > <button class="bton get-intouch form-button" type="submit">Get It Touch</button></li>
```

Comment:
- [ ] Please make your **textarea** field `required` because there should be a message in the email which would be sent to you :+1: